.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+---
+version: 2
+ updates:
+   - package-ecosystem: "docker"
+     directory: "/"
+     schedule:
+       interval: "daily"
```

Comment:
```suggestion
      reviewers:
         - "k3s-io/k3s-dev"
     schedule:
       interval: "monthly"
```

Reply:
Yeah you will want to setup the devs to be added as reviewers. Additionally, for these smaller repos, reducing the turnover is nicer so we don't get swamped by a bunch of tiny PRs.